core/extract/extractor.py

```diff
@@ -156,7 +156,7 @@ def save_logs(
 
         logger.info(f"Generating {len(missing_audit_log_connections)} missing connections.")
         for missing_audit_log_connection_info in missing_audit_log_connections:
-            connection = audit_log_parser.ConnectionLog(
+            connection = ConnectionLog(
```

Comment:
Fix for using connectionLog from `connection_parser.py` and removed `audit_logs_parsing`